app/lib/search/search_service.dart

```diff
@@ -20,6 +22,12 @@ const int resultsPerPage = 10;
 /// links from page 5 to page 15.
 const int maxPages = 10;
 
+/// The tags that we can detect in the user-provided search query.
+final _detectedTags = <String>{
+  ...PackageTags.$all.expand((s) => [s, '-$s', '+$s']),
+  ...PackageVersionTags.$all.expand((s) => [s, '-$s', '+$s']),
```

Comment:
I don't know that we need to support the `+is:legacy` syntax, given that we haven't assigned it any semantics beyond `is:legacy`, maybe it's best we don't?
(not sure where we landed on this last time, either way also okay following that and keeping it).

Reply:
IIRC we have agreed to keep it (as power users will likely expect it to work that way).